specifyweb/frontend/js_src/lib/components/WbPlanView/helpers.ts

```diff
@@ -81,7 +85,7 @@ export async function savePlan({
       expectedResponseCodes: [Http.NO_CONTENT],
     }
   ).then(async () =>
-    newlyAddedHeaders.length === 0
+    newlyAddedHeaders.length === 0 && !hasDeletedLines
```

Comment:
Is it possible to refactor this code to instead check if original line count != new line count?
A check that does computation like this is generally far less bag prone than a flag that you have to remember to check (the hasDeletedLines flag)